packages/image_picker/image_picker_for_web/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: image_picker_for_web
 description: Web platform implementation of image_picker
 repository: https://github.com/flutter/packages/tree/main/packages/image_picker/image_picker_for_web
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+image_picker%22
-version: 2.2.0
+version: 3.0.0
 
```

Comment:
Let's update the `image_picker_platform_interface` to `^2.9.0` so that this can't be used with a version that hasn't deprecated the methods you removed.